src/Client.js

```diff
@@ -265,33 +287,39 @@ export default class Client extends EventTarget {
 
   /**
    * Create a new private payload for the passed in entity, private to the passed in alias.
-   * @param {string} aliasName the name of the alias which is creating the payload.
-   * @param {string} entityId the [id]{@link BaseModel#id} of the entity ({@link Message}, {@link Alias}, or {@link Account}) the payload is attached to.
-   * @param {string} payload the payload to attach to the entity associated with the passed in entityId, private to the alias associated with the passed in alias name.
+   * @param {Object} options
+   * @param {string} options.aliasName the name of the alias which is creating the payload.
+   * @param {string} options.entityId the [id]{@link BaseModel#id} of the entity ({@link Message}, {@link Alias}, or {@link Account}) the payload is attached to.
+   * @param {string} options.payload the payload to attach to the entity associated with the passed in entityId, private to the alias associated with the passed in alias name.
    * @returns {Promise<PrivatePayload>} the new private payload.
    */
-  createPrivatePayload(aliasName, entityId, payload) {
+  createPrivatePayload({aliasName, entityId, payload} = {}) {
```

Comment:
all `xxxPrivatePayload` methods will be moved to the base model. I'm leaving them here for now to keep this PR simple